### ISSUE TITLE
Please install the libjpeg development package before building

### ISSUE THREAD

**Issue by Sev3rus** [imagemin/imagemin-webp#16]
My libjpeg-devel version is 1.2.0.
I run my npm with imagemin-webp, but have this error. Could you help me?
I have done my best for it
`
Error: JPEG support not compiled. Please install the libjpeg development package before building.
Error! Could not process file /tmp/c7048e23-4296-4ddf-bab8-154b1f84e6c4
Error! Cannot read input picture file '/tmp/c7048e23-4296-4ddf-bab8-154b1f84e6c4'
`

**Comment by inakianduaga** [comment]
Same thing for me, failing even after installing `libjpeg libjpeg-devel` on a docker amazon linux image

**Comment by stot3** [comment]
What versions of node are you running?

**Comment by nguyenthetoan** [comment]
@stot3 hi, I'm having the same issue and my node version is now 10.15.3, could you please help me?
the error message:
```
JPEG support not compiled. Please install the libjpeg development package before building.
Error! Could not process file 'xxx'
```

**Comment by nguyenthetoan** [comment]
I just removed my `node_modules` folder and it works fine now 🎉

**Comment by XIOLog** [comment]
On my Ubuntu system I installed package:

`sudo apt-get install libglu1`

It was problem with libGLU.so.1

**Comment by FelixLeChat** [comment]
> I just removed my `node_modules` folder and it works fine now 🎉

Thanks @nguyenthetoan , that fisex the issue for me too

**Comment by geonanorch** [comment]
Possibly `cwebp-bin` [issue #39](https://github.com/imagemin/cwebp-bin/issues/39) can explain the behavior above:
  - if the classical format dev libs (png, jpeg, ...) are not installed _before_ `cwebp-bin` installation is triggered (it is a dependency of `imagemin-webp`), _and_ (for whatever reason) that installation goes on to compile, then support will be missing. Adding those dev libs afterwards will not change that
  - removing `node_modules` probably triggers another recompile in `cwebp-bin`, but this time the dependent libraries are available for compilation and support is added.
  - there is also the special case where there is no compile involved but the dev libs are required at runtime (shared libraries). There adding those libs to the path should be enough to fix it.

**Long story short**: looks like a `cwebp-bin` issue and a workaround might be:
- ensure that the dev library for classical formats (png, jpeg, tiff, dig) are installed
- remove `node_modules/cwebp-bin`
- run `npm i`

**Comment by xstable** [comment]
I had the same issue at an docker-image with linux alpine. 
I solved it by installing following packages: 
```
libjpeg 
jpeg-dev 
libjpeg-turbo-dev 
```

**Comment by httpiga** [comment]
I had this issue too on a Docker Linux image, I solved following the cwep Google guide: https://developers.google.com/speed/webp/docs/compiling#preparing_the_platform

TLDR: `apt-get install -y libjpeg-dev libpng-dev libtiff-dev libgif-dev`

**Comment by ahmedkhan847** [comment]
any facing issue after installing the packages  `apt-get install -y libjpeg-dev libpng-dev libtiff-dev libgif-dev libwebp-dev` remove your node_modules and re install packages it will work

**Comment by LenaNoskova88** [comment]
у меня такая же ошибка Error: JPEG support not compiled. Please install the libjpeg development package before building.
на mac как исправить

**Comment by LenaNoskova88** [comment]
в мене така ж помилка Error: JPEG support not compiled. Please install the libjpeg development package before building.
я на mac  як це виправити

**Comment by sulfab** [comment]
Hello,

I use the library on a Mac Book Pro M3 (ios 14) and i face the issue. 
It seems that the version 1.3.2 of cwebp works but not 1.2.x and 1.1.x version.

I have this error :
`
ERROR in Error with 'static/media/xxx.521ff10e9173ca9b5984.jpg': JPEG support not compiled. Please install the libjpeg development package before building.
` 

I tries to clean node_modules and re install project, but nothing works. 

Anyone is facing this issue ?
